### PR TITLE
feat: publication UI changes

### PR DESCRIPTION
**Pull request by RobCattonKPMG**

### CODE REVIEW

File: site/pages/create-disruption/[disruptionId].page.tsx [hunk 1]
```diff
@@ -437,78 +498,22 @@ const CreateDisruption = (props: DisruptionPageProps): ReactElement => {
...
-                                role="button"
-                                href={`${queryParams["return"] as string}/${pageState.disruptionId}`}
-                                className="govuk-button mt-8 ml-5 govuk-button--secondary"
-                            >
-                                Cancel Changes
-                            </Link>
-                        ) : null}
-                        <button
+                    {displayCancelButton && pageState.disruptionId ? (
+                        <Link
```
Comment (alvak1) [inline site/pages/create-disruption/[disruptionId].page.tsx:509]: The cancel button does not align with Save button

**Review by alvak1 — APPROVED** [review]
Approved based on confirmation that design team have approved the existing state

File: site/pages/create-disruption/[disruptionId].page.tsx (hunk 1 above)
Comment (RobCattonKPMG) [reply site/pages/create-disruption/[disruptionId].page.tsx:509]: Fixed

File: site/pages/create-disruption/[disruptionId].page.tsx [hunk 2]
```diff
@@ -299,56 +298,63 @@ const CreateDisruption = (props: DisruptionPageProps): ReactElement => {
...
+                                    disabled={false}
+                                    disablePast={false}
+                                    inputName={"disruptionStartDate"}
+                                    stateUpdater={validityStateUpdater}
+                                    initialErrors={pageState.errors}
+                                    reset={addValidityClicked}
+                                    schema={validitySchema.shape.disruptionStartDate}
+                                />
+                            </div>
+                            <div className="pl-5">
```
[the remark this answers is not in the document]
Comment (RobCattonKPMG) [reply site/pages/create-disruption/[disruptionId].page.tsx]: Aligned them within a couple of pixels, confirmed its close enough.

**Pull request merged**